sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -267,6 +267,11 @@ func resourceSumologicMonitorsLibraryMonitor() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{"Logs", "Metrics"}, false),
 			},
 
+			"evaluation_delay": {
```

Comment:
if this will be returned in the response from the server by default, you might also want to put `Computed: true` here so the terraform state doesn't show a diff after apply - but if you get a chance you should try it out first and see if it's needed

Reply:
I ran a test where i created a monitor on longData (prod dep). It created successfully. What diff was expected?
See below output:
`sumologic_monitor.test_log_static_resultcount: Creating...
sumologic_monitor.test_log_static_resultcount: Creation complete after 2s [id=000000000000ADAB]`

Reply:
Did you mean that even if i don't specify the field in request, does the response always have it?

Reply:
if you manually create the monitor with this version of the provider and try to do a `terraform apply` after a successful terraform apply, does terraform say there are changes to apply, or does it say no changes detected? if no changes detected then you're fine

Reply:
I added "Computed: true" and also there are no changes to apply once the monitor has been created with the 1st `terraform apply`